Document/0x06b-Basic-Security-Testing.md

```diff
@@ -178,10 +178,6 @@ In order to analyze iOS apps, you should install the following tools on your hos
 
 [Frida-ios-dump](https://github.com/AloneMonkey/frida-ios-dump "frida-ios-dump") allows you to pull a decrypted IPA from a jailbroken device. Please refer to the section "Using Frida-ios-dump" for detailed instructions on how to use it.
 
-##### Ghidra
```

Comment:
i assume this is removed because this is for reverse-engineering right?

Reply:
yes, it was just moved to 6c :)